src/psyclone/psyGen.py

```diff
@@ -415,10 +415,13 @@ class Invoke(object):
     ''' Manage an individual invoke call '''
 
     def __str__(self):
-        return self._name+"("+str(self.unique_args)+")"
+        return self._name+"("+", ".join([str(arg) for arg in
```

Comment:
This line is not pep8-compliant (sorry).

Reply:
What is the error message? My pylint does not complain:
```
joerg@localhost:~/work/psyclone/src/psyclone> pylint ./psyGen.py 
No config file found, using default configuration
************* Module psyclone.psyGen
...
R:337, 4: Too many branches (13/12) (too-many-branches)
E:467,20: Method 'schedule' has no 'calls' member (no-member)
R:421, 4: Too many branches (15/12) (too-many-branches)
```

Reply:
Oh, my mistake! I thought the lack of space around operators would be a problem but maybe it isn't because it's an argument to `return`? Ignore me :-)